src/drivers/pmw3901/pmw3901.cpp

```diff
@@ -38,6 +38,9 @@
  * Driver for the pmw3901 optical flow sensor connected via SPI.
  */
 
+//#define FLOW_SUM
```

Comment:
Can you remove these defines and remove the down sampling. If you see improvements with this I think it's best to remove it from the driver.